app/views/spree/user_sessions/new.html.erb

```diff
@@ -1,16 +1,68 @@
-<% @body_id = 'login' %>
-<div class="container user-sessions user-sessions-new">
-  <div class="row">
-    <div class="col-lg-6 offset-lg-3">
-      <div class="mb-3">
-        <div id="existing-customer" data-hook="login">
-          <%= render partial: 'spree/shared/login' %>
+ <!-- <%# @body_id = 'login' %>
+ <div class="col-md-6 <%= request.path == spree.login_path ? "col-md-offset-3" : "" %>">
+  <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title"><%= Spree.t(:login_as_existing) %></h3>
+    </div>
+      <div id="existing-customer" class="panel-body" data-hook="login">
+          <%#if flash[:alert] %>
+            <div class="alert alert-danger"><%= flash[:alert] %></div>
+          <% #end %>
+          <%#= render :partial => 'spree/shared/login' %>
+          <div class="text-center">
+            <%#= Spree.t(:or) %> 
+                <%#= link_to Spree.t(:create_a_new_account), spree.signup_path %> |
+                <%#= link_to Spree.t(:forgot_password), spree.recover_password_path %>
+          </div>
           <div data-hook="login_extras"></div>
-        </div>
-        <div class="col-lg-11 mx-auto" data-hook="registration">
-          <%= render partial: 'spree/shared/registration', locals: { registration_button: 'registration-button' } %>
-        </div>
       </div>
-    </div>
   </div>
-</div>
+</div>  -->
+<% @body_id = 'login' %>
+<main class="page-contents">
+   <div class="signin-wrapper">
+      <div class="row no-gutters">
+         <div class="col-lg-6">
+            <div id="existing-customer"  data-hook="login">
+               <%= render :partial => 'spree/shared/login' %> 
```

Comment:
The default spree flash msg div is missing. Please add above  this partial: <%= render :partial => 'spree/shared/login' %> 


<%#if flash[:alert] %>
            <div class="alert alert-danger"><%= flash[:alert] %></div>
          <% #end %>